core/src/main/java/org/infinispan/remoting/rpc/RpcManager.java

```diff
@@ -48,6 +48,26 @@
    Map<Address, Response> invokeRemotely(Map<Address, ReplicableCommand> rpcs, RpcOptions options);
 
    /**
+    * Asynchronously sends the {@link ReplicableCommand} to the destination using the specified {@link DeliverOrder}.
+    *
+    * @param destination  the destination's {@link Address}.
+    * @param command      the {@link ReplicableCommand} to send.
+    * @param deliverOrder the {@link DeliverOrder} to use.
+    */
+   void sendTo(Address destination, ReplicableCommand command, DeliverOrder deliverOrder);
```

Comment:
remove ```ResponseMode.ASYNCHRONOUS```? I don't think so. 
The ```invokeRemotely()``` are still useful for replicated caches and tx caches. But, IMO, it doesn't make sense to use them in triangle because we know that all messages are sent asynchronously. It is a overkill to go through the ```invokeRemotely()``` logic. 
We can't do the same for replicated/tx caches the configuration can change the message sending.